src/core/xref.js

```diff
@@ -39,7 +45,16 @@ export async function run(conf, elems) {
     },
   });
   const cache = new IDBKeyVal(idb, "xrefs");
+
+  normalizeConfig(conf);
   const { xref } = conf;
+  if (xref.specs)
```

Comment:
Maybe we can make this a bit simpler:
``` js
if (xref.specs) {
  const bodyCite = document.body.dataset.cite ? document.body.dataset.cite.split(/\s+/) || [];
  document.body.dataset.cite = bodyCite.concat(xref.specs).join(" ");
}
```
(I'm not liking the `getAttribute()` inside a `setAttribute()`)

Reply:
Agree. Good suggestion.

Reply:
Yeah, sorry about that. This is much cleaner, thanks.